shared/src/main/scala/drt/shared/Api.scala

```diff
@@ -219,6 +220,18 @@ case class ApiFlightWithSplits(apiFlight: Arrival, splits: Set[Splits], lastUpda
     this.copy(lastUpdated = None) == candidate.copy(lastUpdated = None)
   }
 
+
+  def isDCSplitsExists =  splits.find(s => s.source == SplitSources.ApiSplitsWithHistoricalEGateAndFTPercentages).isDefined
```

Comment:
1) this could be a val so it's not evaluated every time it's referenced
2) i don't think we want the 'is' prefix - rather `dcSplitsExist`?

Reply:
3. using `split.exists` rather than `splits.find` followed by `isDefined` would be a little more concise